src/lib/components/DeckGLMap/layers/wells/wellsLayer.ts

```diff
@@ -99,7 +99,7 @@ export interface WellsPickInfo extends PickInfo<Feature | LogCurveDataType> {
 }
 
 export default class WellsLayer extends CompositeLayer<
-    Feature | LogCurveDataType,
```

Comment:
Used 'unknown' type for data object of WellsLayer, as in layerTools.tsx pathLayerProps function uses Layer<unknown,P> type and it's causing type incompatibility.